jenkins/run_policies.py

```diff
@@ -55,14 +57,24 @@ def get_custodian_policies(type: str = None):
 
+global_policies = ['empty_buckets', 'empty_roles']
+region = 'us-east-1'
+for policy in global_policies:
+    os.system(f"podman run --rm --name cloud-governance -e account='PERF-DEPT' -e policy={policy} -e AWS_ACCESS_KEY_ID={AWS_ACCESS_KEY_ID_PERF} -e AWS_SECRET_ACCESS_KEY={AWS_SECRET_ACCESS_KEY_PERF} -e AWS_DEFAULT_REGION={region} -e dry_run=yes -e SENDER_MAIL={SENDER_MAIL} -e SENDER_PASSWORD={SENDER_PASSWORD} -e policy_output=s3://{BUCKET_PERF}/{LOGS}/{region} -e log_level=INFO quay.io/ebattat/cloud-governance:latest")
+    os.system(f"podman run --rm --name cloud-governance -e account='PSAP' -e policy={policy} -e AWS_ACCESS_KEY_ID={AWS_ACCESS_KEY_ID_PSAP} -e AWS_SECRET_ACCESS_KEY={AWS_SECRET_ACCESS_KEY_PSAP} -e AWS_DEFAULT_REGION={region} -e dry_run=yes -e SENDER_MAIL={SENDER_MAIL} -e SENDER_PASSWORD={SENDER_PASSWORD} -e policy_output=s3://{BUCKET_PSAP}/{LOGS}/{region} -e log_level=INFO quay.io/ebattat/cloud-governance:latest")
+    os.system(f"podman run --rm --name cloud-governance -e account='RH-PERF' -e policy={policy} -e AWS_ACCESS_KEY_ID={AWS_ACCESS_KEY_ID_RH_PERF} -e AWS_SECRET_ACCESS_KEY={AWS_SECRET_ACCESS_KEY_RH_PERF} -e AWS_DEFAULT_REGION={region} -e dry_run=yes -e SENDER_MAIL={SENDER_MAIL} -e SENDER_PASSWORD={SENDER_PASSWORD} -e policy_output=s3://{BUCKET_RH_PERF}/{LOGS}/{region} -e log_level=INFO quay.io/ebattat/cloud-governance:latest")
+    os.system(f"podman run --rm --name cloud-governance -e account='PERF-SCALE' -e policy={policy} -e AWS_ACCESS_KEY_ID={AWS_ACCESS_KEY_ID_DELETE_PERF_SCALE} -e AWS_SECRET_ACCESS_KEY={AWS_SECRET_ACCESS_KEY_DELETE_PERF_SCALE} -e AWS_DEFAULT_REGION={region} -e SENDER_MAIL={SENDER_MAIL} -e SENDER_PASSWORD={SENDER_PASSWORD} -e dry_run=yes -e policy_output=s3://{BUCKET_PERF_SCALE}/{LOGS}/{region} -e log_level=INFO quay.io/ebattat/cloud-governance:latest")
+    os.system(f"podman run --rm --name cloud-governance -e account='PERF-DEPT' -e policy={policy} -e AWS_ACCESS_KEY_ID={AWS_ACCESS_KEY_ID_DELETE_PERF} -e AWS_SECRET_ACCESS_KEY={AWS_SECRET_ACCESS_KEY_DELETE_PERF} -e AWS_DEFAULT_REGION={region} -e dry_run=no -e SENDER_MAIL={SENDER_MAIL} -e SENDER_PASSWORD={SENDER_PASSWORD} -e log_level=INFO quay.io/ebattat/cloud-governance:latest")
+    os.system(f"podman run --rm --name cloud-governance -e account='PERF-SCALE' -e policy={policy} -e AWS_ACCESS_KEY_ID={AWS_ACCESS_KEY_ID_DELETE_PERF_SCALE} -e AWS_SECRET_ACCESS_KEY={AWS_SECRET_ACCESS_KEY_DELETE_PERF_SCALE} -e AWS_DEFAULT_REGION={region} -e dry_run=no -e SENDER_MAIL={SENDER_MAIL} -e SENDER_PASSWORD={SENDER_PASSWORD} -e log_level=INFO quay.io/ebattat/cloud-governance:latest")
+
```

Comment:
Why we need to run twice each policy with dry run yes and no ?

Reply:
one for dry_run=yes its uploaded to s3
one for dry_run=no, its not uploaded to s3